# Send a SPD (speed) command, linear and angular (m/s), the duration is in s
float32 linear
float32 angular
float32 duration
bool auto_stop
---
bool response